test/integration/svc_int_test.go

```diff
@@ -0,0 +1,26 @@
+package integration
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/svcmanager"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestSvcManagerAndModel(t *testing.T) {
+	cfg, err := config.New()
+	require.NoError(t, err)
+	svcManager := svcmanager.New(cfg)
+	require.NoError(t, svcManager.Start())
+
+	ctx, _ := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Ah, I see.  It is used for testing. :)

Reply:
Yeah we're not really using this particular context atm, the refactor was mainly so that I could use the context for `NewSvcModel()` appropriately. In the previous version we were reusing it for all client requests which will almost certainly eventually trigger a timeout in a way that it shouldn't.